deeplearning4j-nn/src/main/java/org/deeplearning4j/util/NetworkUtils.java

```diff
@@ -0,0 +1,63 @@
+package org.deeplearning4j.util;
+
+import org.deeplearning4j.nn.conf.ComputationGraphConfiguration;
+import org.deeplearning4j.nn.conf.InputPreProcessor;
+import org.deeplearning4j.nn.conf.MultiLayerConfiguration;
+import org.deeplearning4j.nn.conf.NeuralNetConfiguration;
+import org.deeplearning4j.nn.graph.ComputationGraph;
+import org.deeplearning4j.nn.multilayer.MultiLayerNetwork;
+import org.nd4j.linalg.api.ndarray.INDArray;
+
+public class NetworkUtils {
+
+    private NetworkUtils(){ }
+
+    /**
+     * Convert a MultiLayerNetwork to a ComputationGraph
+     *
+     * @return ComputationGraph equivalent to this network (including parameters and updater state)
+     */
+    public static ComputationGraph toComputationGraph(MultiLayerNetwork net){
+
+        //We rely heavily here on the fact that the topological sort order - and hence the layout of parameters - is
+        // by definition the identical for a MLN and "single stack" computation graph. This also has to hold
+        // for the updater state...
+
+        ComputationGraphConfiguration.GraphBuilder b = new NeuralNetConfiguration.Builder()
+                .graphBuilder();
+
+        MultiLayerConfiguration origConf = net.getLayerWiseConfigurations().clone();
+
+
+        int layerIdx = 0;
+        String lastLayer = "in";
+        b.addInputs("in");
+        for(NeuralNetConfiguration c : origConf.getConfs()){
+            String currLayer = String.valueOf(layerIdx);
+
+            InputPreProcessor preproc = origConf.getInputPreProcess(layerIdx);
+            b.addLayer(currLayer, c.getLayer(), preproc, lastLayer);
+
+            lastLayer = currLayer;
+            layerIdx++;
+        }
+        b.setOutputs(lastLayer);
```

Comment:
so, both inputs and outputs go by "in"? That's a little confusing, I find.

Reply:
There's a ```lastLayer = currLayer;```  line there. So it should be something like .addInputs("in") and .setOutputs("9")